### PR TITLE
updated .envexample to include new variables

### PR DESCRIPTION
**Pull request by glenda-wee**
TEST_DOMAIN_CHECK, SENTRY_DSN and DOMAIN_CHECK

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "expo-linear-gradient": "~8.2.1",
     "expo-linking": "^1.0.3",
     "expo-permissions": "~9.0.1",
-    "expo-updates": "^0.2.12",
+    "expo-updates": "~0.2.8",
```
Comment (dalsontws) [inline package.json:27]: oops, might have to update your `expo-updates` package to the latest version

**Review by dalsontws — CHANGES_REQUESTED** [review]

**Review by ilhaoni — CHANGES_REQUESTED** [review]
Haha thanks for adding the example! Do you wanna add a short note into Readme to explain the vars as well? Also remember to remove the package.json updates haha

File: README.md [hunk 2]
```diff
@@ -37,10 +37,20 @@ EXPO_CLI_PROD_USERNAME
 EXPO_CLI_PROD_PASSWORD
 EXPO_CLI_TEST_USERNAME
 EXPO_CLI_TEST_PASSWORD
-SENTRY_AUTH_TOKEN
-SENTRY_DSN
-SENTRY_ORG
-SENTRY_PROJECT
+
+TEST_DOMAIN_CHECK
```
Comment (ilhaoni) [inline README.md:41]: We currently don't use this var, can remove haha

File: README.md [hunk 3]
```diff
@@ -37,10 +37,20 @@ EXPO_CLI_PROD_USERNAME
...
+
+You will also need to define the following environment variables in a .env file (please refer to .env.example):
+
+```
+SENTRY_ORG: the slug of the organisation to use for a command
+SENTRY_PROJECT: the slug of the project to use for a command
+SENTRY_AUTH_TOKEN: the authentication token to use for all communication with Sentry
+SENTRY_DSN: the DSN to use to connect to sentry
+
+DOMAIN_CHECK: to validate backend endpoints
```
Comment (ilhaoni) [inline README.md:53]: Haha should be `DOMAIN_FORMAT`

**Review by ilhaoni — COMMENTED** [review]
Reviewed~ some small changes to be made then ok le! ^^

**Review by ilhaoni — APPROVED** [review]
LGTM! :D thanks for updating our Readme haha

**Comment by ilhaoni** [comment]
@dalsontws need your review again! Haha ^^

**Comment by glenda-wee** [comment]
> LGTM! :D thanks for updating our Readme haha

Thanks for helping me check through! :)

**Comment by glenda-wee** [comment]
> LGTM! :D thanks for updating our Readme haha

Thanks for helping me look through it! :)

**Review by dalsontws — APPROVED** [review]
LGTM!🚢

**Pull request merged**